CMakeLists.txt

```diff
@@ -568,7 +568,7 @@ if (CMAKE_SYSTEM_NAME MATCHES "(SunOS|Solaris)")
 endif ()
 
 if (APPLE AND NOT IOS)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=default")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=default -std=c++11")
```

Comment:
We require C++14 now, but like I said in my previous comment, there must be a different issue going on here, setting the standard is handled by CMake.

I will need a full build log.